models/src_twilio.yml

```diff
@@ -8,9 +8,9 @@ sources:
     loader: fivetran
     loaded_at_field: _fivetran_synced
       
-    freshness:
-      warn_after: {count: 72, period: hour}
-      error_after: {count: 168, period: hour}
+    freshness: 
+      warn_after: {count: 72, period: hour} 
+      error_after: {count: 168, period: hour} 
```

Comment:
Was there any specific reason these freshness tests were aligned on for **all** source tables? I imagine this may not be consistent with the freshness requirements of all tables. What are your thoughts?

Reply:
Good question. With this I just referenced other packages like klaviyo and iterable. But I suppose it doesn't make sense for example for account_history to use this frequent of a freshness test. I therefore moved it to call, message, and usage_record and changed it to a 24-hour warning, based on what I see in the sample customer data

Reply:
Let me know what you think!

Reply:
I think this is a good approach. However, can we maybe adjust the 24 hour warn window to 72 instead. Just to give a little bit more buffer.

Finally, be sure to add the [disable config for sources](https://github.com/fivetran/dbt_zendesk_source/blob/8b78b0d3a02d168abbf1afefed4cf7e114dad313/models/src_zendesk.yml#L94-L95) that can be disabled (like calls in this case)